src/main/java/io/xpring/XpringKitException.java

```diff
@@ -2,7 +2,15 @@
 
 /** Exceptions which occur when working with XpringKit. */
 public class XpringKitException extends Exception {
-    public XpringKitException(String errorMessage) {
-        super(errorMessage);
+    /** Static exception for when a classic address is passed to an X-Address API. */
```

Comment:
FWIW - I still think you should move to a sub-package of `io.xpring` so your'e not squatting on the package (for future code). E.g., this project should probably be `io.xpring.xpring4j` maybe.